### PR TITLE
add new VOC blog post

### PR DESCRIPTION
**Pull request by bgruening**

### CODE REVIEW

File: _posts/2021-05-11-B-1-617-2-detected.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+site: [freiburg]
+title: "April Galactic News!"
+tags: 
+- galaxy
+- COVID-19
+- elixir
```
Comment (beatrizserrano) [inline _posts/2021-05-11-B-1-617-2-detected.md:7]: supporter instead of tag?

File: _posts/2021-05-11-B-1-617-2-detected.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+---
+
+With B.1.617.2 (one of the Indian lineages) now being a variant of concern (VOC), we checked the COG-UK data that we have been analyzing to
```
Comment (beatrizserrano) [inline _posts/2021-05-11-B-1-617-2-detected.md:10]:
```suggestion
With B.1.617.2 (one of the Indian lineages) now being a Variant Of Concern (VOC), we checked the COG-UK data that we have been analyzing to
```

**Comment by nekrut** [comment]
This should not be just an .eu blog. It should be galaxywide one

**Comment by bgruening** [comment]
We are actually crosslinking those blogs. But we can also put them on ORG first and corsslink it back here. Is the text ok?

**Comment by nekrut** [comment]
coordinate with @tnabtaf so it is in sync with galaxyproject.org

**Comment by tnabtaf** [comment]
Make it live on EU.  Once it is up there, I will cross link from ORG.

**Comment by beatrizserrano** [comment]
ok, then I'll merge, thanks @tnabtaf

**Pull request merged**